modules/core/src/test/java/org/apache/ignite/internal/metric/CacheMetricsAddRemoveTest.java

```diff
@@ -204,12 +253,14 @@ private void checkMetricsEmpty(String cachePrefix) {
 
```

Comment:
`MetricRegistry mreg = mmgr.registry(cachePrefix);`

Can we just iterate over mmgr and check that there is no registry with specified name? Mentioned above call creates registry if it does not exist. I see no need for this.

Reply:
I think this is not related to the issue as well as other refactoring.